tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/external/langtags.json

```diff
@@ -42462,23 +42447,10 @@
         "regionname": "Greece",
         "script": "Grek",
         "sldr": false,
-        "tag": "grc-Grek",
+        "tag": "grc",
+        "tags": [ "grc-GR", "grc-Grek" ],
```

Comment:
no - please don't change langtags.json!  it's an external file (hence `/external/` … ) -  fyi @mhosken @DavidLRowe @emily-roth

Reply:
No worries. I also submitted a PR to their project https://github.com/silnrsi/langtags/pull/21

But I don't have to get ahead of myself, I'll undo the json changes. I wasn't sure whether to be this aggressive or not.

Reply:
it's better to override with code or other data than to modify external files- makes it hard to reproduce the situation